templates/taxi/car_detail.html

```diff
@@ -11,12 +12,21 @@ <h1>
       Update
     </a>
   </h1>
+  
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  
   <h1>Drivers</h1>
   <hr>
   <ul>
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  
+  {% if user in car.drivers.all %}
+    <a href="{% url 'taxi:remove-from-car' car.id %}" class="btn btn-warning">Delete me from this car</a>
```

Comment:
To implement this feature thorugh post method of CarDetailView you will need to change links to  form with different inputs and different values that you will later compare in your view. Also you can include a hidden field here with car_id so you can pull it out in post method and use to create links between car and driver